intel-retail/automated-self-checkout
# Release3 cleanup

## PR Checklist



## What are you changing?

Refactoring code to only include content related to the automated self checkout use case. Performance tool submodule has been added to provide benchmark features.

## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable


- Make run-demo
- See video stream with full pipline running. This includes object detection, text detection, text recognition, and barcode detection.

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
